index.html

```diff
@@ -0,0 +1,666 @@
+                        <li>Logout</li>
+                    </ul>
+                </div>
+                <div class="col-2"></div>
+            </div>
+        </nav>
+
+        <!--Lower navigation-->
+        <nav>
+            <div class="row lower_navigation">
+                <div class="col-1"></div>
+                <div class="col-2 about_logo">
+                    <img src="images/nmc 2.webp" alt="">
+                </div>
+                <div class="col-1"></div>
+                <div class="col-8">
+                    <ul>
+                        <li>
+                            <a href="about.html">About</a>
+                        </li>
+                        <li>
+                            <a href="##">Program</a>
+                        </li>
+                        <li>
+                            <a href="##">Join</a>
+                        </li>
+                        <li>
+                            <a href="##">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="##">News</a>
+                        </li>
+                        <li>
+                            <a href="##">Follow us</a>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+        </nav>
+
+        <!--main section-->
+        <div class="container-fluid main-section"
+            style="background-image:url('https://fevenseyfu.github.io/Javascript-Hackathon/assets/images/Hero-background.png');background-repeat:no-repeat;background-size:cover;margin:0px 0px !important;">
+            <div class="container hambuger">
+                <div class="row">
+                    <div class="col-1 hambuger-menu">
+                        <img src="./images/mobile_menu.png" alt="menu-bar">
+
+                    </div>
+                </div>
+
+                <!--Hamburger drop down-->
+                <div class="container-fluid hide-menu">
+                    <div class="row">
+                        <div class="col-12 drop-down-menu">
+
+                            <ul>
+                                <li>
+                                    <a href="about.html">About</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
```

Comment:
[OPTIONAL]
- [ ] It would be nicer if once the user hits a navigation link, they are redirected to that specific section on the page :smile: